website/app.py

```diff
@@ -21,8 +22,12 @@
 import website.models
 from website.routes import make_url_map
 from website.addons.base import init_addon
-from website.project.model import ensure_schemas
+from website.project.model import ensure_schemas, Node
 
+def build_js_config_files(settings):
+    with open(os.path.join(settings.STATIC_FOLDER, 'public', 'js', 'nodeCategories.js'), 'wb') as fp:
+        fp.write("window.contextVars.nodeCategories = {0}".format(dumps(Node.CATEGORY_MAP)))
```

Comment:
@sloria You suggested we put built JS files in /public... is there a require() path for public files?